gh2/csv.py

```diff
@@ -207,24 +209,26 @@ def set_headers(repo, labels=None):
     return headers
 
 
-def main():
-    parser = make_parser()
+def get_all_labels(repositories):
```

Comment:
@git-harry it would also require change to the ``issue_to_dict`` function which also uses ``additional_labels`` as a list of label objects, but I take the point about the label objects from different repos not being the same even if they are the same name, so I'll try and work it out